--- a/src/teensy/powershell_shellcode.py
+++ b/src/teensy/powershell_shellcode.py
@@ -149,7 +149,7 @@ Keyboard.set_modifier(0);
 Keyboard.set_key1(0);
 Keyboard.send_now();
 }
-""" % (core.powershell_encodedcommand())
+""" % (core.powershell_encodedcommand()))
 print("[*] Payload has been extracted. Copying file to {0}".format(os.path.join(core.setdir + "reports/teensy.ino")))
 if not os.path.isdir(os.path.join(core.setdir + "reports")):
     os.makedirs(os.path.join(core.setdir + "reports"))
